doc/manuals/user/forbidden_characters.md

```diff
@@ -77,19 +77,19 @@ Let's consider the following `notification.httpCustom` object in a given subscri
 "httpCustom": {
```

Comment:
The original example was based in the "JSON as text" case, which no longer makes sense with the new `json` field functionality. Thus, it has been slightly adapted.

NTC (just informative)